### ISSUE TITLE
Error Local variable in global space caused by mod.cpp syntax

### ISSUE THREAD

**Issue by jonpas** [toadie2k/NIArms#4]
As noted on ACE3's public Slack.

NIArms Core v1.0.

The following in `mod.cpp`:
```
hidePicture = "false";
hideName = "false";
```
causes:
```
20:11:49 Global namespace not passed during: false
20:11:49 Global namespace not passed during: false
20:11:49 Error in expression <false>
20:11:49   Error position: <false>
20:11:49   Error Local variable in global space
```
on game start.

Fix is:
```
hidePicture = 0;
hideName = 0;
```

**Comment by theYoursTruly** [comment]
Mod.cpp is currently not on the GitHub. Maybe it should be added? @toadie2k

**Comment by toadie2k** [comment]
resolved in b385501ca4859e2bccff39ba2331c9b5ed778b76
